export/export_handler.go

```diff
@@ -93,7 +93,7 @@ func (h *Handler) generateTasks(startData StartData, config catalog.ExportConfig
 			diffs, hasMore, err = getDiffFromBase(context.Background(), startData.Repo, startData.ToCommitRef, after, limit, h.cataloger)
 		} else {
 			// TODO(guys) change this to work with diff iterator once it is available outside of cataloger
```

Comment:
this comment is outdated then?

Reply:
Not yet, 
we still need to paginate over cataloger results instead of using an iterator.
will be outdated once we could get an iterator from the cataloger (or once we decide we don't want to support it).